### PR TITLE
added labels to multiple resources in VS ontology

### PR DESCRIPTION
**Pull request by Bardawulf**
Signed-off-by: Bardawulf <kanarek365@gmail.com>

Fixes: #126

### CODE REVIEW

**Comment by mereolog** [comment]
@Bardawulf is this ready for review?
If so, could you remove the WIP suffix from the name of this pull request?

File: VS/VehicleSignals.rdf [hunk 1]
```diff
@@ -2290,6 +2290,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Intake-AirTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Intake air temperature sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:2293]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 2]
```diff
@@ -2336,6 +2337,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;IsBackUpLightOn">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Is backup light on</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:2340]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 3]
```diff
@@ -2954,6 +2956,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;LeftIndicatorLight">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Left indicator light</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:2959]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 4]
```diff
@@ -2964,6 +2967,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Lidar">
 		<rdfs:subClassOf rdf:resource="&sosa;Sensor"/>
+		<rdfs:label xml:lang="en">Lidar</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:2970]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 5]
```diff
@@ -3185,6 +3189,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;LumbarPressureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Lumbar pressure sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:3192]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 6]
```diff
@@ -4024,6 +4030,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;RightIndicatorLight">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Right indicator light</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:4033]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 7]
```diff
@@ -4365,6 +4372,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;Sonar">
 		<rdfs:subClassOf rdf:resource="&sosa;Sensor"/>
+		<rdfs:label xml:lang="en">Sonar</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:4375]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 8]
```diff
@@ -4687,6 +4695,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;ThrottleActuator">
 		<rdfs:subClassOf rdf:resource="&sosa;Actuator"/>
+		<rdfs:label xml:lang="en">Throttle actuator</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:4698]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

File: VS/VehicleSignals.rdf [hunk 9]
```diff
@@ -5083,6 +5092,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;VoltageSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Voltage sensor</rdfs:label>
```
Comment (mereolog) [inline VS/VehicleSignals.rdf:5095]: Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.

**Review by mereolog — CHANGES_REQUESTED** [review]

**Review by mereolog — APPROVED** [review]

**Review by mskulim — APPROVED** [review]

**Pull request merged**